### DISCUSSION TITLE
Remove Enentlistener from zoom

### DISCUSSION THREAD

**Discussion by NuraSad** [rpearce/react-medium-image-zoom#553] [Q&A] [answered]
How can i remove eventlisteners from zoom function and make it unzoom only by button click?

**Comment by rpearce** [comment, marked as answer]
Look in the README and stories examples for the Controlled component. You can determine when it zooms and unzooms, and the change handler will let you know when it "should" zoom or unzoom.

I'm not currently sending back the event in the zoom change callback, but I wouldn't rule out doing so if necessary (that would allow you to check what the currentTarget is and check if it's tagName is a button).

Sorry for formatting--typing quickly on the mobile app!

**Reply by NuraSad** [reply]
So, can  I use onZoomChange to remove eventListeners like "wheel" from zoom() function(as I understood zoom() add them by default)? If so, can you help me to understand this. In CustomControl example onZoomChange just equals undefined.

**Reply by rpearce** [reply]
@NuraSad You can't remove any of the listeners while the component is mounted, but you can choose to ignore their default behavior if zooming and unzooming, if you wish, by using the `Controlled` component.
